src/Bicep.LangServer/Handlers/BicepHoverHandler.cs

```diff
@@ -192,7 +192,7 @@ private static async Task<MarkedStringsOrMarkupContent> GetModuleMarkdown(HoverP
                 }
             }
 
-            var descriptions = string.Join("\n", descriptionLines.WhereNotNull().ToArray());
+            var descriptions = string.Join("\n\n", descriptionLines.WhereNotNull().ToArray());
```

Comment:
Che ToArray() call can be removed to avoid allocation.

Reply:
494982f2209b6c15698e8fd6a7a7ebe2ca016346

Thanks, in TypeScript, an enumerable isn't allowed there.